src/js/button.js

```diff
@@ -27,64 +24,65 @@ var Button = Component.extend({
     this.on('focus', this.onFocus);
     this.on('blur', this.onBlur);
   }
-});
-
-Component.registerComponent('Button', Button);
 
-Button.prototype.createEl = function(type, props){
-  // Add standard Aria and Tabindex info
-  props = Lib.obj.merge({
-    className: this.buildCSSClass(),
-    'role': 'button',
-    'aria-live': 'polite', // let the screen reader user know that the text of the button may change
-    tabIndex: 0
-  }, props);
-
-  let el = Component.prototype.createEl.call(this, type, props);
-
-  // if innerHTML hasn't been overridden (bigPlayButton), add content elements
-  if (!props.innerHTML) {
-    this.contentEl_ = Lib.createEl('div', {
-      className: 'vjs-control-content'
-    });
-
-    this.controlText_ = Lib.createEl('span', {
-      className: 'vjs-control-text',
-      innerHTML: this.localize(this.buttonText) || 'Need Text'
-    });
-
-    this.contentEl_.appendChild(this.controlText_);
-    el.appendChild(this.contentEl_);
+  createEl(type, props) {
+    // Add standard Aria and Tabindex info
+    props = Lib.obj.merge({
+      className: this.buildCSSClass(),
+      'role': 'button',
+      'aria-live': 'polite', // let the screen reader user know that the text of the button may change
+      tabIndex: 0
+    }, props);
+
+    let el = Component.prototype.createEl.call(this, type, props);
```

Comment:
Yeah, I didn't do the most exhaustive search of old super method use cases, except for in the constructor. I guess I should do that.